src/libs/Wallet/index.ts

```diff
@@ -0,0 +1,18 @@
+import type {Card} from '@src/types/onyx';
+
+function checkIfWalletIsAvailable(): Promise<boolean> {
+    return Promise.resolve(false);
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function handleAddCardToWallet(_card: Card, _cardHolderName: string, _onFinished?: () => void) {
+    // no op for other platforms
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function isCardInWallet(_card: Card) {
+    // Return true for other platforms, so the AddToWalletButton is always hidden
+    return Promise.resolve(true);
```

Comment:
Is this necessary? It looks like src/libs/Wallet/RNAddToWalletButton/index.tsx already doesn't return the button so it shouldn't show up regardless